backend/src/apiserver/server/test_util.go

```diff
@@ -29,29 +29,80 @@ import (
 	"github.com/stretchr/testify/assert"
 	"google.golang.org/grpc/codes"
 	authorizationv1 "k8s.io/api/authorization/v1"
+	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
 	invalidPipelineVersionId = "not_exist_pipeline_version"
 )
 
-var testWorkflow = util.NewWorkflow(&v1alpha1.Workflow{
+// var testWorkflow = util.NewWorkflow(&v1alpha1.Workflow{
+// 	TypeMeta:   v1.TypeMeta{APIVersion: "argoproj.io/v1alpha1", Kind: "Workflow"},
+// 	ObjectMeta: v1.ObjectMeta{Name: "workflow-name", UID: "workflow1", Namespace: "ns1"},
+// 	Spec:       v1alpha1.WorkflowSpec{Arguments: v1alpha1.Arguments{Parameters: []v1alpha1.Parameter{{Name: "param1"}}}},
+// })
+
+// var testWorkflow2 = util.NewWorkflow(&v1alpha1.Workflow{
+// 	TypeMeta:   v1.TypeMeta{APIVersion: "argoproj.io/v1alpha1", Kind: "Workflow"},
+// 	ObjectMeta: v1.ObjectMeta{Name: "workflow-name", UID: "workflow2"},
+// 	Spec:       v1alpha1.WorkflowSpec{Arguments: v1alpha1.Arguments{Parameters: []v1alpha1.Parameter{{Name: "param1"}}}},
+// })
+
+// var testWorkflowPatch = util.NewWorkflow(&v1alpha1.Workflow{
+// 	TypeMeta:   v1.TypeMeta{APIVersion: "argoproj.io/v1alpha1", Kind: "Workflow"},
+// 	ObjectMeta: v1.ObjectMeta{Name: "workflow-name", UID: "workflow2"},
+// 	Spec:       v1alpha1.WorkflowSpec{Arguments: v1alpha1.Arguments{Parameters: []v1alpha1.Parameter{{Name: "param1"}, {Name: "param2"}}}},
+// })
```

Comment:
nit: clean up?

Reply:
sorry for being so sloppy, working on to many things at once.